odkx-src/sync-endpoint.rst

```diff
@@ -89,12 +89,13 @@ ODK-X Sync Endpoint does not store user information in its own database, instead
 
 HTTPS
 -----
+  HTTPS stands for Hyper Text Transfer Protocol Secure. It is a protocol for securing the communication between two systems e.g. the browser and the web server. To know more about HTTPS and its working have a look at this video `Working of HTTPS <https://www.youtube.com/watch?v=T4Df5_cojAs>`_.
 
   The Sync Endpoint stack integrates support for automatic certificate
   provisioning via domain validation and letsencrypt. For most use
   cases this should be sufficient. Certificate provisioning parameters
   can be edited interactively during initialization or directly in
-  :file:`config/https.env`.
+  :file:`config/https.env`. To learn about cert bot and letsencrypt visit this site `Cert bot <https://docs.min.io/docs/generate-let-s-encypt-certificate-using-concert-for-minio.html>`_.
```

Comment:
Is cert bot one word certbot? And is letsencrypt one word? Or should it have caps or anything?

Reply:
Yes Certbot and letsencrypt are one words. I have made the correction.